apps/fluent-tester/src/FluentTester/testPages.macos.ts

```diff
@@ -128,6 +129,11 @@ export const tests: TestDescription[] = [
     component: TabsTest,
     testPage: HOMEPAGE_TABS_BUTTON,
   },
+  {
+    name: 'Tabs Experimental Test',
+    component: TabsExperimentalTest,
```

Comment:
Most tests prefix "Experimental" (I.E: Experimental Button), while this one makes it a suffix. I prefer your style, but could we keep it consistent across them alll?

Reply:
Sure! Consistency is probably best.